Dockerfile.prod

```diff
@@ -0,0 +1,19 @@
+FROM node:12
+
+ENV HOST 0.0.0.0
+ENV NODE_ENV=development
+
+RUN yarn install
+
+ENV NODE_ENV=production
+
+WORKDIR /opt/acompanha_legis
+
+COPY . .
+
+RUN yarn build
```

Comment:
Thinking on docker best practices, I think that we can remove some unnecessary files after the yarn build and other stuffs from COPY above just to reduce the image size.
The folder created by yarn build is enough, isn't it?

Reply:
I think I can work further on it... but not only a folder I think, check here: https://nuxtjs.org/guide/commands/

Since it's SSR, it will also run a server under 3000 to server the files correctly (`yarn start`), this is slightly different from regular `SPA` with totally "compiled" static files.

Reply:
I'll check which are the necessary folder and add the rest to `.dockerignore`, is it ok?